src/fortran/client.F90

```diff
@@ -232,14 +234,30 @@ function SR_error_parser(self, response_code) result(is_error)
 end function SR_error_parser
 
 !> Initializes a new instance of a SmartRedis client
-function initialize_client(self, cluster)
-  integer(kind=enum_kind)           :: initialize_client
-  class(client_type), intent(inout) :: self    !< Receives the initialized client
-  logical, optional,  intent(in   ) :: cluster !< If true, client uses a database cluster (Default: .false.)
+function initialize_client(self, cluster, logger_name)
+  integer(kind=enum_kind)                     :: initialize_client
+  class(client_type),         intent(inout)   :: self      !< Receives the initialized client
+  logical, optional,          intent(in   )   :: cluster   !< If true, client uses a database cluster (Default: .false.)
+  character(len=*), optional, intent(in   )   :: logger_name !< Identifier for the current client
+
+  ! Local variables
+  character(kind=c_char, len=:), allocatable :: c_logger_name
+  integer(kind=c_size_t) :: logger_name_length
+
+  if (present(logger_name)) then
+    allocate(character(kind=c_char, len=len_trim(logger_name)) :: c_logger_name)
+    c_logger_name = logger_name
+    logger_name_length = len_trim(logger_name)
+  else
+    allocate(character(kind=c_char, len=10) :: c_logger_name)
```

Comment:
Doesn't really matter, but the length here should be 7

Reply:
used len_trim to eliminate hard-coded lengths